Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -1,903 +0,0 @@
-    get { unexpectedBetweenColonAndPlatforms }
-    set { unexpectedBetweenColonAndPlatforms = newValue }
-  }
-
-  @available(*, deprecated, renamed: "platforms")
-  var versionList: AvailabilityVersionRestrictionListSyntax {
-    get { platforms }
-    set { platforms = newValue }
-  }
-
-  @available(*, deprecated, renamed: "unexpectedAfterPlatforms")
-  var unexpectedAfterVersionList: UnexpectedNodesSyntax? {
-    get { unexpectedAfterPlatforms }
-    set { unexpectedAfterPlatforms = newValue }
-  }
-
-  /// Adds the provided `element` to the node's `versionList`
-  /// collection.
-  /// - param element: The new `Availability` to add to the node's
-  ///                  `versionList` collection.
-  /// - returns: A copy of the receiver with the provided `Availability`
-  ///            appended to its `versionList` collection.
-  @available(*, deprecated, renamed: "addPlatform")
-  func addAvailability(_ element: AvailabilityVersionRestrictionListEntrySyntax) -> BackDeployedAttributeSpecListSyntax {
-    addPlatform(element)
-  }
-
-  @available(*, deprecated, message: "Use an initializer with a platforms argument")
-  init(
-    leadingTrivia: Trivia? = nil,
-    _ unexpectedBeforeBeforeLabel: UnexpectedNodesSyntax? = nil,
-    beforeLabel: TokenSyntax = .keyword(.before),
-    _ unexpectedBetweenBeforeLabelAndColon: UnexpectedNodesSyntax? = nil,
-    colon: TokenSyntax = .colonToken(),
-    _ unexpectedBetweenColonAndVersionList: UnexpectedNodesSyntax? = nil,
-    versionList: AvailabilityVersionRestrictionListSyntax,
-    _ unexpectedAfterVersionList: UnexpectedNodesSyntax? = nil,
-    trailingTrivia: Trivia? = nil
-  ) {
-    self.init(
-      leadingTrivia: leadingTrivia,
-      unexpectedBeforeBeforeLabel,
-      beforeLabel: beforeLabel,
-      unexpectedBetweenBeforeLabelAndColon,
-      colon: colon,
-      unexpectedBetweenColonAndVersionList,
-      platforms: versionList,
-      unexpectedAfterVersionList,
-      trailingTrivia: trailingTrivia
-    )
-  }
-}
-
-public extension DeclGroupSyntax {
-  @available(*, deprecated, renamed: "memberBlock")
-  var members: MemberDeclBlockSyntax {
-    get {
-      return memberBlock
-    }
-    set(value) {
-      memberBlock = value
-    }
-  }
-}
-
-public extension EditorPlaceholderDeclSyntax {
-  @available(*, deprecated, renamed: "placeholder")
-  var identifier: TokenSyntax { placeholder }
-
-  @available(*, deprecated, renamed: "placeholder")
-  @_disfavoredOverload
-  init(
-    leadingTrivia: Trivia? = nil,
-    _ unexpectedBeforeIdentifier: UnexpectedNodesSyntax? = nil,
-    identifier: TokenSyntax,
-    _ unexpectedAfterIdentifier: UnexpectedNodesSyntax? = nil
-  ) {
-    self.init(
-      leadingTrivia: leadingTrivia,
-      unexpectedBeforeIdentifier,
-      placeholder: identifier,
-      unexpectedAfterIdentifier
-    )
-  }
-}
-
-public extension DictionaryTypeSyntax {
-  @available(*, deprecated, renamed: "unexpectedBeforeLeftSquare")
```

Comment:
Is this also missing from the generated compatibility layer?

Reply:
Forgot to add the deprecated names